include/lsst/meas/base/FlagHandler.h

```diff
@@ -39,6 +39,15 @@ namespace lsst { namespace meas { namespace base {
     C-strings so we can create these arrays using initializer lists even in C++98.
  */
 struct FlagDefinition {
+
+    FlagDefinition() {
+    }
+
+    FlagDefinition(char const * _name, char const * _doc) {
+        name = _name;
+        doc = _doc;
+    }
+
```

Comment:
Good catch.  I am not sure about the lifetime issue here, but it certainly should be thought about.  Doesn't the same issue come put for the original addFields() call?  Seems like what protects those pointers is that they are attached to a static member of the class which is using them.  That means that someone abusing the original addFields() could probably create the same issue.  But I propose that I adopt the same strategy with Python, and use a static list for the flagDefs and only call the addFields from the plugin constructor.